packages/space-opera/src/components/materials_panel/materials_panel.ts

```diff
@@ -125,12 +137,65 @@ export class MaterialPanel extends ConnectedLitElement {
     this.safeUrlIds = safeUrlIds;
   }
 
+  /* Interpolate from red to original color as n approaches maxN */
+  getInterpolatedColor(original: RGBA, n: number, maxN: number): RGBA {
+    const red: RGBA = [1, 0, 0, 1];
+    const percentRed = (maxN - n) / maxN;
```

Comment:
It's only a percent if you multiply it by 100; I'd call this a ratio or fraction.